blueprints/glimmer-component/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   },
 
   locals(options) {
-    let className = stringUtils.classify(options.entity.name);
+    let className = `${stringUtils.classify(options.entity.name)}Component`;
```

Comment:
not sure I'm a fan of this, as it brings to mind imagines of ye olde global resolver :P